scipy/linalg/decomp.py

```diff
@@ -343,24 +376,35 @@ def eigh(a, b=None, lower=True, eigvals_only=False, overwrite_a=False,
     Raises
     ------
     LinAlgError
-        If eigenvalue computation does not converge,
-        an error occurred, or b matrix is not definite positive. Note that
-        if input matrices are not symmetric or Hermitian, no error is reported
-        but results will be wrong.
+        If eigenvalue computation does not converge, an error occurred, or
+        b matrix is not definite positive. Note that if input matrices are
+        not symmetric or Hermitian, no error will be reported but results will
+        be wrong.
 
     See Also
     --------
     eigvalsh : eigenvalues of symmetric or Hermitian arrays
     eig : eigenvalues and right eigenvectors for non-symmetric arrays
-    eigh : eigenvalues and right eigenvectors for symmetric/Hermitian arrays
     eigh_tridiagonal : eigenvalues and right eiegenvectors for
         symmetric/Hermitian tridiagonal matrices
 
     Notes
     -----
     This function does not check the input array for being hermitian/symmetric
     in order to allow for representing arrays with only their upper/lower
-    triangular parts.
+    triangular parts. Also, note that even though not taken into account,
+    finiteness check applies to the whole array and unaffected by "lower"
+    keyword.
+
+    This function uses LAPACK drivers for computations in all possible keyword
+    combinations, prefixed with ``sy`` if arrays are real and ``he`` if
+    complex. As a brief summary, the slowest and the most robust driver is the
```

Comment:
Maybe mention that ``g`` and ``e`` are the next part of the prefix so that the remainder of the explanation distinguishes among gv/gvd/gvx, too?

Reply:
Hmm. I guess you are right. I'll clarify